script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -195,16 +197,16 @@ I get the feeling that this boy definitely knows what I should be doing from her
 [sha:49e90d033c74fadaf73e8585782ac7707aa8d18c]{
 -- Page 686, Offset 26113.
 -- 「あの、代行？　わたしもですか？　っていうか、座って休んでいいですか？」
-"Um, Executor, sir? May I come as well? Or rather, can I sit down and rest?"
+"Um, Presbyter, sir? May I come as well? Or rather, can I sit down and rest?"
 }
 [sha:01fa531daa17053a238e1b04addba4713b498d27]{
 -- Page 686, Offset 26114.
 -- 「死ね、ブタ。テメェ俺の後ろでＳＰだろうが。ベンチに座んのはオレだけだ。トオノはそこまで気を許しちゃいねえからな、どうせそこで立ちん坊だろ？」
 // Don't think SP is going to be understood by the audience directly here
-"Kill yourself, fatso.
+"Kill yourself, Fatso.
```

Comment:
why is fatso capitalized in these files

Reply:
I think the reasoning is as a nickname? Not sure I agree with it either though